engine/apps/alerts/models/channel_filter.py

```diff
@@ -68,6 +70,16 @@ class ChannelFilter(OrderedModel):
 
     created_at = models.DateTimeField(auto_now_add=True)
     filtering_term = models.CharField(max_length=1024, null=True, default=None)
```

Comment:
could we get away with having just the `filtering_term` field?

Store `filtering_term_jinja2` inside of `filtering_term` and use the value of `filtering_term_type` to figure out whether or not it's regex or jinja

Reply:
yep, thank you, I was not using `filtering_term_type` in the first prototype

I had to add a bit more complicated logic to default values on the frontend, but worth of it